docs/adr/adr-0021-from-rem-to-em.md

```diff
@@ -0,0 +1,111 @@
+---
+kind: '📌 Docs/Architecture Decision Records'
+---
+# ADR 0021: Switching from rem to em units
+
+🗓️ 2022-08-29 · ✍️ Florian Sanders
+
+## The context
+
+Since the beginning of this component library, we've been using relative units for almost everything in our CSS.
+
+### Using relative units for font size
+
+Users can set their base font size through the Operating System settings and the browser settings as well.
+However, this setting may be completely ignored by some browsers when the website font sizes have been set using fixed units like `px`. 
+This is the case with Chrome for instance, where font sizes in `px` remain the same whatever base font size you select in the settings.
+
+Users can still Zoom, but they would have to do it on every single website they are viewing.
+Usually, when people truly need bigger characters, they want to specify the size once for instance from the Operating System settings. They expect it to be applied everywhere, including the browser interface and every website they visit.
+
+### Using relative units for everything else
+
+Using relative font sizes is not enough because when the text gets bigger, containers should also get bigger.
+If not, then the text will overflow and may very well become unreadable.
```

Comment:
I feel like the "very well" is not necessary

Reply:
You're right, I'll remove it :+1: